Automattic/mShots
# Add performance testing script to mshots

We have plans which will put a significantly increased load on mshots, I'm adding these scripts to test whether mshots is up to the task and also to have a way to test future performance improvements.

I went with a python script using the [locust](https://docs.locust.io/en/stable/quickstart.html#quickstart) framework because we've used python for one off things around wpcom and I think the framework is quite nice.

p2 link discussing this: pbxlJb-1sw-p2

#### Test instructions

Follow instructions in [scripts/performance-testing/README.md ](https://github.com/Automattic/mShots/pull/61/files#diff-ec6dc62d3438d1428c824b1ca558539d1e684acf1b8bde888458689d3c5a3b5d) and ensure that they capture everything that needs to be done on your machine.
